bindings/python/setup.py

```diff
@@ -34,13 +31,14 @@ def get_extension_modules():
         if os.name == 'posix':
             module.extra_compile_args.append('-std=c++11')
 
-    return arrow_modules + libbson_modules
+    return modules
 
 
 setup(
     name='pymongoarrow',
     version=get_pymongoarrow_version(),
     packages=find_packages(),
     ext_modules=get_extension_modules(),
-    install_requires=['pyarrow >= 3', 'pymongo >= 3.11,<4'],
+    install_requires=['pyarrow >= 3', 'pymongo >= 3.11,<4', 'pandas',
```

Comment:
While this is fine for now, I think eventually `pandas` and `numpy` need to be in `tests_require`.
Setuptools has shifted to using `pyproject.toml` and `setup.cfg` instead of our current setup: https://setuptools.readthedocs.io/en/latest/userguide/quickstart.html

Reply:
Ok, but what if a user only wants to use pymongoarrow with numpy? I don't think they should be forced to install pandas.

Reply:
The way to support that would be to put pandas and numpy in extras_require.